save-backend/src/main/kotlin/com/saveourtool/save/backend/service/TestsSourceVersionService.kt

```diff
@@ -122,11 +123,9 @@ class TestsSourceVersionService(
     }
 
     private fun doDelete(versionEntity: TestsSourceVersion) {
-        // clean-up [TestsSuite]
-        testSuitesService.deleteTestSuite(versionEntity.snapshot.source, versionEntity.name)
         versionRepository.delete(versionEntity)
         val snapshotEntity = versionEntity.snapshot
-        if (versionRepository.findAllBySnapshot(snapshotEntity).isEmpty()) {
+        if (versionRepository.findAllBySnapshotId(snapshotEntity.requiredId()).isEmpty()) {
```

Comment:
There is no difference, the search is still carried out by ID.

Reply:
ok, do you suggest return back to search by entity?

Reply:
No need. Just saying for the future :)